config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-registry.yml

```diff
@@ -0,0 +1,50 @@
+periodics:
```

Comment:
this is largely a copy of the existing rclone based job

it will run in dry-run mode initially